px-data-grid-paginated.html

```diff
@@ -317,19 +319,20 @@
             },
 
             /**
-             * Default column width if not defined, eg. '100px'
+             * Default column width if not defined, eg. '100px'. Notice that with not
+             * zero flex values this value behaves more as minimum width. See
+             * defaultColumnFlexGrow property.
              */
             defaultColumnWidth: {
-              type: String,
-              value: '100px'
+              type: String
             },
 
             /**
-             * Default column flex if not defined, eg. 1
+             * Default column flex if not defined, eg. 1. Equialent to the CSS flex-grow
+             * property.
              */
-            defaultColumnFlex: {
-              type: Number,
-              value: 1
```

Comment:
why the default value was removed?

Reply:
Actually, ignore my earlier commit, it's two way binding so this will use the default of px-data-grid. No need to duplicate default.s